vendors/swedbank-pay-core-library/src/SwedbankPay/Core/Order/PlatformUrls.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace SwedbankPay\Core\Order;
+
+use SwedbankPay\Core\Data;
+
+/**
+ * Class PlatformUrls
+ * @package SwedbankPay\Core\Order
+ * @method string getCompleteUrl()
+ * @method string getCancelUrl()
+ * @method string getCallbackUrl()
+ */
+class PlatformUrls extends Data implements PlatformUrlsInterface
+{
+    /**
+     * PlatformUrls constructor.
+     *
+     * @param array $data
+     */
+    public function __construct(array $data = [])
+    {
+        $this->setData($data);
+    }
+
+    /**
+     * Get urls where hosts
+     *
+     * @return array
+     */
+    public function getHostUrls()
+    {
+        $urls = [];
+
+        foreach ($this->getData() as $key => $url) {
+            if (filter_var($url, FILTER_VALIDATE_URL, FILTER_FLAG_SCHEME_REQUIRED)) {
+                $parsed = parse_url($url);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function parse_url() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)